### ISSUE TITLE
请问是否需要集成  kmod-shortcut-fe-cm？

### ISSUE THREAD

**Issue by ghost** [gwlim/mips24k-lede-patch#2]
请问是否需要集成  kmod-shortcut-fe-cm？
还是只需要集成kmod-fast-classifier kmod-shortcut-fe就可以了？

感谢！

**Comment by bardimnik** [comment]
THANKS!!! YES!!!
Select all the 3 modules: kmod-fast-classifier, kmod-shortcut-fe, kmod-shortcut-fe-cm
THANKS!!!

**Comment by bardimnik** [comment]
Hello! I'm glad I found you. I have long wanted to get rid of the stock firmware on the TP-Link Archer C7 v1. But I was stopped by the fact that in the OpenWRT / LEDE there was no support for "Hardware Nat". When I found you, I decided to compile the firmware myself, but I have little knowledge of how to correctly do it. I tried to do it myself, but I did not succeed. Errors occurred during compilation. Apparently I have something not been installed.
I ask you to include in the hardware nat (kmod-fast-classifier kmod-shortcut-fe kmod-shortcut-fe-cm). And more, if possible, turn on the firmware to connect a USB flash drive. With the possibility of expanding the system to LEDE. THANKS YOU!!!
